delta_go/render.py

```diff
@@ -0,0 +1,94 @@
+from pathlib import Path
+
+import numpy as np
```

Comment:
Would add a docstring explaining that this file just renders the game visually so they don't need to understand it in full